core/consensus/babe/src/lib.rs

```diff
@@ -586,23 +586,21 @@ fn check_header<B: Block + Sized, C: AuxStore>(
 			if check(&inout, threshold) {
 				match check_equivocation(&client, slot_now, slot_num, header.clone(), signer.clone()) {
```

Comment:
`check_equivocation(bla).map_err(ToString::to_string)?`

Will remove some writing below in the match.

The same can probably be done for aura.